.github/workflows/pr-check.yml

```
@@ -0,0 +1,20 @@
+name: PR Builder & Converage Check
+
+on:
+  pull_request:
+    branches: [ main, dev ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)